api/CSSKeyframeRule.json

```diff
@@ -28,31 +43,81 @@
             },
             {
               "version_added": "5",
-              "prefix": "moz"
+              "version_removed": "48",
+              "prefix": "Moz"
             }
           ],
           "ie": {
             "version_added": "10"
           },
-          "opera": {
-            "version_added": "12",
-            "prefix": "o"
-          },
-          "opera_android": {
-            "version_added": true
-          },
-          "safari": {
-            "version_added": "4"
-          },
-          "safari_ios": {
-            "version_added": true
-          },
-          "samsunginternet_android": {
-            "version_added": "5.0"
-          },
-          "webview_android": {
-            "version_added": "45"
-          }
+          "opera": [
+            {
+              "version_added": "18"
+            },
+            {
+              "version_added": "15",
+              "version_removed": "18",
+              "prefix": "WebKit"
+            },
+            {
+              "version_added": "12",
+              "version_removed": "15"
+            }
+          ],
+          "opera_android": [
+            {
+              "version_added": "18"
+            },
+            {
+              "version_added": "14",
+              "version_removed": "18",
+              "prefix": "WebKit"
+            },
+            {
+              "version_added": "12",
+              "version_removed": "14"
+            }
+          ],
+          "safari": [
+            {
+              "version_added": "9.1"
```

Comment:
The change was in https://trac.webkit.org/changeset/176157/webkit, at WebKit trunk version 601.1.10. Based on that I would have expected it to be in Safari 9. It looks like we have test results for 9.1 but not 9 in mdn-bcd-results. Does the script incorrectly infer 9.1, or was this a range version where you removed the range?

Reply:
In BrowserStack and SauceLabs, we unfortunately don't have a way to differentiate 9 and 9.1 in testing.  (This is probably a good reason to get more testing VMs.) I think I determined the version number from reading through the documentation, but I can bump it down to Safari 9!

Reply:
I see. As a reviewer, when this uncertainty exists due to a lack of browser versions to test, it would be great if it's made explicit that it's a guess. Where it's recent enough or there are other entries I'd expect it to be in sync with, I might then hunt down the WebKit commit to confirm which of the two versions seems more likely.

Let's go with "9" here.